File: frontend/src/api/applications/dto/ResearchAreaDto.ts
export type ResearchAreaDto = {
  id: string;
  name: string;
};

export function getResearchAreaName(initialAreas: ResearchAreaDto[], id: string) {
  for (const area of initialAreas) {
    if (area.id == id) return area.name;
  }
  return null;
}

File: backend/ResearchCruiseApp/Api/Applications/Contracts/ResearchAreaDto.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Serialization;

namespace ResearchCruiseApp.Api.Applications.Contracts;

[JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
public class ResearchAreaDto(Guid id, string name)
{
    public Guid Id { get; set; } = id;

    public string Name { get; set; } = name;
}
